builtin/providers/aws/resource_aws_lightsail_instance.go

```diff
@@ -35,7 +35,12 @@ func resourceAwsLightsailInstance() *schema.Resource {
 			},
 			"blueprint_id": {
 				Type:     schema.TypeString,
-				Required: true,
```

Comment:
This shouldn't really be Optional, so much as requiring exactly one of blueprint_id or instance_snapshot_name. I don't know how to express that. Either that or I should replace this with "image_name" and then detect if it's one of the blueprints or if it's a snapshot. Which might be the more elegant solution.